index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="menu-wrap">
+      <div class="logo">
+        <span class="brand">Saw Mon Han</span>
+        <input type="checkbox" class="toggler" />
+        <div class="hamburger"><div></div></div>
+        <div class="menu">
+          <div class="overlay">
+            <div>
+              <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <main>
+      <article class="intro">
+        <h1 class="pb branding">
+          I'm Saw Mon,
+          <br />
+          Glad to See you!
+        </h1>
+        <p class="pb introduction">
+          I am a software developer! I can help you build a product, feature or
+          website look through some of your work and experience! If you like
+          what you see and have a project you need coded, don't hesitate to
+          contact me.
+        </p>
+        <p class="pb connect">LET'S CONNECT</p>
```

Comment:
- [ ] I think it would be better to use a ```<header>``` tag for this section, as a page header contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. 👍 👍